examples/workflow/main.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/dapr/go-sdk/client"
+	"github.com/dapr/go-sdk/workflow"
+)
+
+var stage = 0
+
+const (
+	workflowComponent = "dapr"
+)
+
+func main() {
+	wr, err := workflow.NewRuntime()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println("Runtime initialized")
+
+	if err := wr.RegisterWorkflow(TestWorkflow); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("TestWorkflow registered")
+
+	if err := wr.RegisterActivity(TestActivity); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("TestActivity registered")
+
+	var wg sync.WaitGroup
+
+	// Start workflow runner
+	fmt.Println("runner started")
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		if err := wr.Start(); err != nil {
```

Comment:
The internal implementation of `Start()` already creates a background goroutine, so I don't think we need to do this call to `Start()` in a goroutine. Instead, just call it directly.